iiblib/iib_client.py

```diff
@@ -87,6 +87,7 @@ def add_bundles(
         overwrite_from_index=False,
         overwrite_from_index_token=None,
         raw=False,
+        deprecation_list=None,
```

Comment:
How about defaulting this to `[]` instead?

Reply:
https://docs.quantifiedcode.com/python-anti-patterns/correctness/mutable_default_value_as_argument.html
This way it would be anti-pattern

Reply:
@midnightercz could you please comment on this?

Reply:
I agree, [] is dangerous default for kwarg

Reply:
Thank you

Reply:
However documentation of this argument would be nice (in doc-string below)

Reply:
The commit `Change an order of deprecation_list var in add_bundle` should be squashed with previous one - no need to have this in single commit